Documentation/ExtensionArchitecture/Extbase/Reference/FrontendPlugins.rst

```diff
@@ -91,27 +91,9 @@ Use the following steps to add the plugin as content element:
 
 #.  (Optional) Add to the :guilabel:`New Content Element` wizard
 
-    Add the following :ref:`page TSconfig <t3tsconfig:setting-page-tsconfig>`
-    to add the new plugin to the wizard:
-
-    ..  literalinclude::  _FrontendPlugin/_page.tsconfig
```

Comment:
This file  is not used anymore and needs to be deleted. Can you do that or should I do it in a follow-up?